test 5/README.txt

```diff
@@ -0,0 +1,2 @@
+This test checks two suppliers supplying the same and one supplier supplying different products
+Very similiar to the previous test, a little bigger though
```

Comment:
looks identical to me. what am I missing?

Reply:
Does seem like it, I will remove it